templates/financial-services/index.html

```diff
@@ -236,6 +246,25 @@ <h2>Reduce IT operations cost</h2>
   </div>
 </section>
 
+<div class="u-fixed-width">
+  <hr class="p-separator"/>
+</div>
+
+<section class="p-strip">
```

Comment:
just wondering if this part starts with a new `section`. 

1. If it's a new section maybe we don't need a separator above and change the class here with `p-strip--light`? 
```suggestion
<section class="p-strip--light">
```
2. If it's part of the section above, remove this `section` tag?  

What do you think?

Reply:
Great idea, I have removed the break and added --light. It looks a lot better now, thanks!